src/views/Recorder/BrowserEventLog.tsx

```diff
@@ -0,0 +1,70 @@
+import { BrowserEvent } from '@/schemas/recording'
+import { exhaustive } from '@/utils/typescript'
+import { FaceIcon } from '@radix-ui/react-icons'
+import { Flex, ScrollArea, Table } from '@radix-ui/themes'
+import { Allotment } from 'allotment'
+
+interface EventIconProps {
+  event: BrowserEvent
+}
+
+function EventIcon({ event }: EventIconProps) {
+  switch (event.type) {
+    case 'dummy':
+      return <FaceIcon />
+
+    default:
+      return exhaustive(event.type)
+  }
+}
+
+interface EventDescriptionProps {
+  event: BrowserEvent
+}
+
+function EventDescription({ event }: EventDescriptionProps) {
+  switch (event.type) {
+    case 'dummy':
+      return (
+        <span>
+          This is a <strong>dummy</strong> event fired at an interval.
+        </span>
+      )
+
+    default:
+      return exhaustive(event.type)
+  }
+}
+
+interface BrowserEventLogProps {
+  events: BrowserEvent[]
+}
+
+export function BrowserEventLog({ events }: BrowserEventLogProps) {
+  return (
+    <Flex direction="column" minHeight="0" height="100%">
+      <Allotment defaultSizes={[1]}>
```

Comment:
Using allotment here because it sets a fixed height on the element and that makes the `ScrollArea` work correctly. Please let me know if you know any better way, but I've tried settings flex here and flex there and it just won't do it for me.